roles/st2/tasks/user.yml

```diff
@@ -10,7 +10,6 @@
     ssh_key_file: "{{ st2_ssh_key_file }}"
     state: present
   register: _user
-  tags: [st2, user]
```

Comment:
I moved `tags` for this role into `tasks/main.yml`, see: https://github.com/StackStorm/ansible-st2/blob/4b9cb5188a34f8cde587a10c57401078c947af51/roles/st2/tasks/main.yml

So tags are applied there, - a bit easier and tags still can be used.